### ISSUE TITLE
Add election config to configuration passed to frontend from election-server

### ISSUE THREAD

**Issue by soerenbf** [Concordium/concordium-governance-committee-voting#141]
To avoid problems with parsing the result returned from the election contract view functions (which are quite large), it would make sense to do this on the backend, and "hydrate" the frontend state with value passed from the backend.

**Comment by abizjak** [comment]
The response will be even bigger (substantially) when it is in JSON and will be sent from the backend to the frontend.

So not clear what problem this is solving. 

If parsing is slow it would make sense to look into why that is. Because it's a simple process.

**Comment by soerenbf** [comment]
> The response will be even bigger (substantially) when it is in JSON and will be sent from the backend to the frontend.
> 
> So not clear what problem this is solving.
> 
> If parsing is slow it would make sense to look into why that is. Because it's a simple process.

True, I'll rephrase this to concern itself with the issue instead of the solution.
